apps/cookbook/src/app/showcase/link-showcase/link-showcase.component.html

```diff
@@ -0,0 +1,45 @@
+<p>
+  Links are used for navigation between pages. They can be standalone links used on their own or
+  inline links placed within a sentence.
+</p>
+<p>
+  A link should be used for simple navigation. Use
```

Comment:
I still think it would be worth mentioning that a link should have a `href` attribute. Otherwise it's most likely a `<button>` and not a link.

Consider if you can merge that point in here somewhere.